articles/diagnose-diabetes-with-svm/index.md

```diff
@@ -0,0 +1,319 @@
+![Hero Header](/engineering-education/diagnose-diabetes-with-svm/HERO.jpg)
+
+# Diagnosis of Diabetes using Support Vector Machines
+       
+In this guide, we will learn how to use machine learning to diagnose if a patient has diabetes. We can do this by using their medical records. We will use the Support Vector Machine Algorithm (from Sci-kit Learn) to build our model. The GitHub repo for this project is [here](http://github.com/Inyrkz/diagnose-diabetes-svm).
+
+## Prerequisite
+- A PC with Jupyter Notebook
+- Basic Python Knowledge
+- Basic knowledge of Support Vector Machines 
+- Diabetes dataset from [Kaggle](https://www.kaggle.com/uciml/pima-indians-diabetes-database)
+
+## Outline
+- Exploratory Data Analysis with Pandas-Profiling
+- Feature Extraction
+- Split Dataset into Training and Test Set
+- Creating the SVM Model
+- Diagnosing a New Patient
+- Assess Model Performance
+
+## EXPLORATORY DATA ANALYSIS WITH PANDAS-PROFILING
+The pandas-profiling library helps us do quick exploratory data analysis with minimal effort. To install pandas-profiling, run the code below:
+
+```python
+pip install pandas-profiling
+```
+
+If you are using Anaconda, then you can run the following code in Anaconda Prompt:
+
+```python
+conda install -c conda-forge pandas-profiling
+```
+
+Now we can import pandas-profiling and generate a report for our dataset. Before we load our dataset, let us import the libraries we will be using.
+
+```python
+# importing libraries
+import numpy as np
+import pandas as pd
+import pandas_profiling
+from sklearn import svm
+from sklearn.preprocessing import StandardScaler
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import accuracy_score, classification_report
+```
+
+We import our dataset using the read_csv function from pandas. We pass the dataset filename as an argument.
+
+```python
+# Getting Data
+dataset = pd.read_csv("diabetes.csv")
+```
+
+```python
+# generate report with panda-profiling
+profile = dataset.profile_report(title='Diabetes Profiling Report')
+profile
+```
+
+Pandas-profiling gives us the dataset statistics.
+
+![Overview](/engineering-education/diagnose-diabetes-with-svm/1.jpg)
+
+From the report above, we have nine variables and 768 rows. There are no missing data in the dataset. There are no duplicate data.
+
+Pandas-profiling also looks at each of the nine variables. For each variable, it gives us descriptive statistics. It generates a histogram that shows the data distribution of each variable.
+
+![Variables](/engineering-education/diagnose-diabetes-with-svm/2.jpg)
+![Variables](/engineering-education/diagnose-diabetes-with-svm/3.jpg)
+![Variables](/engineering-education/diagnose-diabetes-with-svm/4.jpg)
```

Comment:
Please also provide a heading below each image that describes what the image is.
do this for all images that you've included in the article.

Reply:
Done!
Should the image headings use #### or ### ?

Reply:
It's sufficient if it uses normal bolded text

```markdown
**Text**
```